refactor(framework) Rename environment variable to control log level (#4880)
adap/flower: framework/docs/source/how-to-configure-logging.rst

.. note::

    You can make use of the ``FLWR_LOG_LEVEL`` environment variable when executing other
    Flower commands to provision the different components in a Flower Federation (see
    :doc:`how-to-run-flower-with-deployment-engine`) or using the `flwr CLI
    <ref-api-cli.html>`_.

Configure gRPC logging
----------------------

> Flower users `gRPC <https://grpc.io/>`_ to communicate between each component (see
> :doc:`ref-flower-network-communication`). You can set the verbositiy level of ``gRPC``
> logs using `gRPC environment variables
> <https://github.com/grpc/grpc/blob/master/doc/environment_variables.md>`_.

Reviewer: ```suggestion
Flower uses `gRPC <https://grpc.io/>`_ to communicate between each component (see
:doc:`ref-flower-network-communication`). You can set the verbosity level of ``gRPC``
logs using `gRPC environment variables
<https://github.com/grpc/grpc/blob/master/doc/environment_variables.md>`_.
```
